demos/sound/flxsound/source/PlayState.hx

```diff
@@ -0,0 +1,26 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.FlxState;
+import flixel.text.FlxText;
+import flixel.ui.FlxButton;
+import flixel.math.FlxMath;
+
+class PlayState extends FlxState
+{
+	override public function create():Void
+	{
+		bgColor = 0;
+		super.create();
+
+		var soundBox:FlxButton = new FlxButton(0, 0, "Play Sound", playSound);
```

Comment:
Code-style wise, the first type hint here is redundant - it's better style to rely on type inference:

```haxe
var soundBox = new FlxButton(0, 0, "Play Sound", playSound);
```